doc/introduction/inspecting_circuits.rst

```diff
@@ -0,0 +1,208 @@
+ .. role:: html(raw)
+   :format: html
+
+
+.. _intro_inspecting_circuits:
+
+Inspecting circuits
+===================
+
+PennyLane offers functionality to inspect, visualize or analyze quantum circuits.
+
+.. _intro_qtransforms:
+
+Most of these tools are implemented as **transforms**
+which take a :class:`~pennylane.QNode` instance and return a function:
```

Comment:
```suggestion
Most of these tools are implemented as **transforms**.  Transforms take a :class:`~pennylane.QNode` instance and return a function:
```

Not quite sure why I prefer this, but my gut is telling me it sounds a little better.